packages/database/src/modelClasses/EntityHierarchy.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Just a note: I added the `EntityHierarchy` and `EntityRelation` model to database package to allow creating dummy records for project hierarchies which are required for some test cases in GET visualisations

Reply:
This will have conflicts with my upcoming refactor of the entity hierarchy building and database models, but shouldn't be too bad